### ISSUE TITLE
Korean translation of 'If then' contains reporter block syntax '()'

### ISSUE THREAD

**Issue by kg1992** [scratchblocks/scratchblocks#308]
Hello, thanks for an amazing plug-in.

I recently found that it is impossible to write 'if' block in Korean locale. ('ko.json')
Not that the translation is wrong, but the translation includes reporter block syntax.
> if <mouse down> then
is translated into
> 만약 <마우스를 클릭했는가?> (이)라면

They will be looking like [this](http://scratchblocks.github.io/#?style=scratch3&lang=ko&script=%09%09%0A%09%EB%A7%8C%EC%95%BD%20%3C%EB%A7%88%EC%9A%B0%EC%8A%A4%EB%A5%BC%20%ED%81%B4%EB%A6%AD%ED%96%88%EB%8A%94%EA%B0%80%3F%3E%20(%EC%9D%B4)%EB%9D%BC%EB%A9%B4%0A%09%0A%09if%20%3Cmouse%20down%3E%20then).

**Comment by tjvr** [comment]
Oh interesting. Does it appear in Scratch with the brackets?

We pull translations from the same repository that Scratch itself uses, so I'd rather not customise them too much 🤔 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by kg1992** [comment]
Yes, translation in locale file ko.json seems identical to what scratch app shows.

Long answer: They put parenthesis around 이, because the last letter of preceding word decides its appearance, and that is up to which block user inserts there. Thus, no one can predict it.

Since it is not desirable to customize locale file, I might try to solve this issue by modifying locale data in ‘scratchblocks’ object using javascript on fly. If that is possible.

**Comment by tjvr** [comment]
I'm not sure why I didn't suggest this before, but you can write this currently by escaping the bracket:
```
만약 <마우스를 클릭했는가?> \(이)라면
```
Ideally the translator would do this for you, of course...
